packages/RichTextEditing-Tests.package/RichTextEditorImportExportTests.class/instance/testUS87ExportDocumentAsPlainText.st

```diff
@@ -0,0 +1,9 @@
+acceptance-tests
+testUS87ExportDocumentAsPlainText
+
+	self enterStringOntoTextField: self exampleExportContent.
+	self editorModel saveText.
+	self chooseExportStrategy: RichTextPlainTextExportStrategy.
+	
+	self assert: self exportedFileExists.
+	self assert: self exportedFileContent equals: self exampleExportContent
```

Comment:
Maybe split in two tests

Reply:
Have to test if file exists anyway otherwise weird exceptions occur